### ISSUE TITLE
PB625 When running on the screen only interference.

### ISSUE THREAD

**Issue by chelovekPlanetaZ** [koreader/koreader#4476]
* KOReader version: 2019.01
* Device: Pocketbook PB625

#### Issue
When you start it shows what's on the photo. Before this was version 2018.11 everything worked. I decided to update by air (the first time), I thought that the "crooked" was updated then manually updated the result is the same.

![-0037 150x200](https://user-images.githubusercontent.com/45392108/51083322-e7fcaf80-1720-11e9-9e79-d6e993f6831f.jpg)

##### `crash.log` (if applicable)
```
---------------------------------------------
                launching...
  _  _____  ____                _
 | |/ / _ \|  _ \ ___  __ _  __| | ___ _ __
 | ' / | | | |_) / _ \/ _` |/ _` |/ _ \ '__|
 | . \ |_| |  _ <  __/ (_| | (_| |  __/ |
 |_|\_\___/|_| \_\___|\__,_|\__,_|\___|_|

 It's a scroll... It's a codex... It's KOReader!

 [*] Current time: 01/12/19-15:27:18
 [*] Version: v2019.01

ffi.load: blitbuffer
ffi.load (warning): libblitbuffer.so: cannot open shared object file: No such file or directory
ffi.load: SDL2
ffi.load (warning): libSDL2.so: cannot open shared object file: No such file or directory
ffi.load: SDL
ffi.load (warning): libSDL.so: cannot open shared object file: No such file or directory
ffi.load: inkview
01/12/19-15:27:18 INFO  SoftwareVersion:  R625.4.4.1815
PB FB: smem_len    : 487424
PB FB: line_length : 600
PB FB: xres        : 600
PB FB: xres_virtual: 600
PB FB: yres        : 800
PB FB: yres_virtual: 800
PB FB: bpp         : 8
FB: computed size: 480000
FB: id           : 
FB: smem_len     : 487424
FB: line_length  : 600
FB: xres         : 600
FB: xres_virtual : 600
FB: yres         : 800
FB: yres_virtual : 800
FB: bpp          : 8
Atached mp shm: id 0 addr 40026000 size 17868
----------------> USING A2PLUS MODE <-----------------
attach_shm: mtfb key=251331144 size=518336
Atached mtfb shm: id 150008 addr 40703000 size 518336
-------------------------
The task - koreader.app have
 xres=600 yres=800 bpp=4 vdepth=8
-------------------------
ndepth:4 hqdepth:4 orientation:0
attach_shm: ref key=2130378752 size=480000
Atached ref shm: id 8001 addr 40797000 size 480000
OpenTheme: /ebrmain/themes/Contrast.pbt
ts: loading default calibration values
ffi.load: inkview
01/12/19-15:27:18 INFO  initializing for device PocketBook
01/12/19-15:27:18 INFO  framebuffer resolution: {
    ["h"] = 800,
    ["w"] = 600
}
Atached mqueue: id 8001
ffi.load: libs/libfreetype.so.6
ffi.load: libs/libmupdf.so
ffi.load: libs/libwrap-mupdf.so
01/12/19-15:27:19 INFO  opening file ./help/quickstart-en-v2019.01.html
01/12/19-15:27:20 INFO  Loading plugins from directory: plugins
01/12/19-15:27:20 INFO  Plugin  plugins/autofrontlight.koplugin/main.lua  has been disabled.
01/12/19-15:27:20 INFO  Plugin  plugins/autosuspend.koplugin/main.lua  has been disabled.
ffi.load: sqlite3
ffi.load: z
ffi.load: sqlite3
01/12/19-15:27:20 INFO  Plugin  plugins/hello.koplugin/main.lua  has been disabled.
01/12/19-15:27:20 INFO  Plugin  plugins/keepalive.koplugin/main.lua  has been disabled.
01/12/19-15:27:20 INFO  Plugin  plugins/kobolight.koplugin/main.lua  has been disabled.
01/12/19-15:27:20 INFO  RD loaded plugin SSH at plugins/SSH.koplugin
01/12/19-15:27:20 INFO  RD loaded plugin backgroundrunner at plugins/backgroundrunner.koplugin
01/12/19-15:27:20 INFO  RD loaded plugin batterystat at plugins/batterystat.koplugin
01/12/19-15:27:20 INFO  RD loaded plugin calibrecompanion at plugins/calibrecompanion.koplugin
01/12/19-15:27:20 INFO  CoverBrowser: setting default display modes
01/12/19-15:27:21 INFO  RD loaded plugin coverbrowser at plugins/coverbrowser.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin evernote at plugins/evernote.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin goodreads at plugins/goodreads.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin kosync at plugins/kosync.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin newsdownloader at plugins/newsdownloader.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin perceptionexpander at plugins/perceptionexpander.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin readtimer at plugins/readtimer.koplugin
01/12/19-15:27:21 INFO  RD loaded plugin send2ebook at plugins/send2ebook.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin statistics at plugins/statistics.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin systemstat at plugins/systemstat.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin terminal at plugins/terminal.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin texteditor at plugins/texteditor.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin timesync at plugins/timesync.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin wallabag at plugins/wallabag.koplugin
01/12/19-15:27:22 INFO  RD loaded plugin zsync at plugins/zsync.koplugin
```

**Comment by Frenzie** [comment]
Also see #4447. Pinging @NiLuJe

**Comment by chelovekPlanetaZ** [comment]
Thank you very much Frenzie. Sorry if something is wrong.

**Comment by NiLuJe** [comment]
Okay, I *explicitly* did my maths with 600x800 because I was expecting that one.

Which means there's a stupid typo somewhere, yay.

@chelovekPlanetaZ : No, you did nothing wrong, that was exactly what we needed ;).

**Comment by chelovekPlanetaZ** [comment]
Thank you very much NiLuJe works as it should!
> No, you did nothing wrong, that was exactly what we needed ;).
Glad to help ;).

**Comment by NiLuJe** [comment]
Oops, forgot to close this ;).

Fixed in #4479 ;).
